dashboard/app/models/concerns/text_to_speech.rb

```diff
@@ -5,7 +5,7 @@
 require 'redcarpet'
 require 'redcarpet/render_strip'
 
-TTS_BUCKET = 'cdo-tts'.freeze
+TTS_BUCKET = 'cdo-tts'.freeze unless defined? TTS_BUCKET
```

Comment:
Arguably needing to do the `unless defined?` is an indication that we shouldn't be polluting the global namespace with global constants. Unlike python, this will be a global "variable" and could conflict with TTS_BUCKET defined by some random library or whatever.

Not a must-fix to merge by any means, progress is progress, but if inclined I think this is only referenced outside this class by  `bin/oneoff/migrate_tts_files.rb` and could move it into the class (all untested):

```
class TTSSafe < Redcarpet::Render::StripDown
  TTS_BUCKET = 'cdo-tts'.freeze
...
```

And then reference it in migrate_tts_files.rb as `TTSSafe::TTS_BUCKET`.

Reply:
makes sense! done. when I try to run the script from the repo root, it appears to be broken:
```
./bin/oneoff/migrate_tts_files.rb:49:in `require_relative': cannot load such file -- /Users/dsb/src/cdo/bin/oneoff/dashboard/config/environment (LoadError)
```
if I fix the require statement, I'm able to verify that `TextToSpeech::TTS_BUCKET` is accessible and has the right value.

Reply:
I also tend to find dead scripts when I do big refactors: I think we have a bunch of them